docs/faq/lukso/feature-requests.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_label: 'Feature Requests'
+sidebar_position: 3
+---
+
+# Feature Requests
+
+### Will there be video guides about LSPs?
+
+Yes, LUKSO is planning to create videos for LSPs both on the theoretical and practical end. They will be released after the mainnet process, and the LSP development is finalized with audits and grant programs.
+
+### Will there be re-occurring developer events?
+
+Yes, developer meetings and events are planned to present our monthly tech progress and move into a more transparent development process. You can now find and discuss weekly tech updates within the official [LUKSO Discord](https://discord.gg/lukso) server.
+
+### Will there be another LUKSO Hackathon?
+
+Yes, we’re planning and preparing future hackathon projects. For official announcements, please follow our [LUKSO Twitter](https://twitter.com/lukso_io) account. You can already have a look into the developer channels on our [LUKSO Discord](https://discord.gg/lukso) in order to find teammates or project ideas.
+
+### Is there going to be a grants program?
+
+Yes, We're preparing to launch the grant program after the mainnet migration process is fully finalized. As you can see in the [LYX distribution](https://medium.com/lukso/the-lyxe-migration-process-374053e5ddf5), there are reserves and the foundation's treasury to support grants and drive the development of the network.
+
+### Will there be DappNode or Avado Support for LUKSO?
+
+We are collaborating with the [DappNode](https://dappnode.com/) team to provide official support for the LUKSO blockchain. The support will likely be available within the launch process. However, we do not have information about potential other node providers adding support.
+
+### Will there be a Ledger integration for LUKSO?
+
+You can use your [Ledger](https://www.ledger.com/de) to store LYXe and LYX already using the Ethereum application. Before the mainnet launch process is not fully completed, there are no active approaches to adding official integrations to hardware wallets. Please note that it is not within our sole control, as external wallet providers decide which blockchains to add.
```

Comment:
```suggestion
You can use your [Ledger](https://www.ledger.com/) to store LYXe and LYX already using the Ethereum application. Before the mainnet launch process is not fully completed, there are no active approaches to adding official integrations to hardware wallets. Please note that it is not within our sole control, as external wallet providers decide which blockchains to add.
```

maybe you can mention that Ledger are used through MetaMask.

Maybe you can add that Ledger may display "Network = Kovan" because of chainId 42, and explain why it shows like this.

Reply:
Note: the kovan thing is about to be fixed anyway so no need to put it here.